### PR TITLE
PO-94 added cron creation steps in config

### PR DESCRIPTION
**Pull request by abdulwahidsharief**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: admin/language/en-gb/extension/payment/razorpay.php [hunk 1]
```diff
@@ -112,6 +112,12 @@
 $_['text_invoice']                                      = 'Invoice Details';
 $_['text_not_select_sub_ID']                             = 'Please Select Subscription ';
 $_['text_subscription_status']                          = 'Enable Subscription in Settings.';
+$_['text_webhook_cron_header']                          = 'Set the cron job in your WHMCS site server to call the Cron URL in every 5 mins frequncy.';
+$_['text_webhook_cron_point_1']                          = 'In CLI run: <strong>crontab -e</strong>';
```
Comment (ramth05) [inline admin/language/en-gb/extension/payment/razorpay.php:116]: @abdulwahidsharief , why we can't use single variable and pass the HMTL within that?

Comment (abdulwahidsharief) [reply admin/language/en-gb/extension/payment/razorpay.php:116]: @ramth05 we can pass. but there will be many html tags inside a single string(<ol>,<li><label>) and cron url creating in controller we need to check how we can bring it in language file

**Review by ramth05 — APPROVED** [review]
LGTM

File: admin/language/en-gb/extension/payment/razorpay.php (hunk 1 above)
Comment (abdulwahidsharief) [reply admin/language/en-gb/extension/payment/razorpay.php]: merged all points as one and using sprintf for cron url

**Pull request merged**